database/migrations/20190305135521-create-articles.js

```diff
@@ -0,0 +1,52 @@
+/* eslint no-unused-vars: ["error", { "args": "none" }] */
+export default {
+  up: (queryInterface, Sequelize) => (
+    queryInterface.createTable('articles', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      title: {
+        type: Sequelize.STRING,
+      },
+      slug: {
+        type: Sequelize.STRING
+      },
+      body: {
+        type: Sequelize.TEXT
+      },
+      description: {
+        type: Sequelize.STRING
+      },
+      // tagList: {
```

Comment:
#### Nitpick
- No commented if it is essential in the future then put a `TODO` comment tag above it explaining why you are commenting it out.

Reply:
Okay. I never knew about the todo comment. I commented it because it was going to be important in the future